### PR TITLE
form update

### PR DESCRIPTION
**Pull request by annetajoy**
- modified Marks
- modified phone/email
- modified Community Form
- modified Management form and added second language ,religion input

### CODE REVIEW

File: src/components/form-elements/Location.jsx [hunk 1]
```diff
@@ -1,42 +1,192 @@
...
-    gramaPanchayath,
-    setGramaPanchayath,
-  } = useContext(DataContext);
-
+  const { state, district, taluk, gramaPanchayath, setGramaPanchayath } =
+    useContext(DataContext);
+  const { Option, OptGroup } = Select;
+  function onChange(value) {
+    console.log(`selected ${value}`);
+  }
```
Comment (vishnuchandrappan) [inline src/components/form-elements/Location.jsx]: - Update this function properly.
- remove `console.log`

File: src/components/form-elements/Location.jsx [hunk 2]
```diff
@@ -1,42 +1,192 @@
...
+          <Option value="Kottayam">Kottayam</Option>
+          <Option value="Kozhikode"> Kozhikode</Option>
+          <Option value="Malappuram">Malappuram</Option>
+          <Option value="Palakkad">Palakkad</Option>
+          <Option value="Pathanamthitta">Pathanamthitta</Option>
+          <Option value="Thiruvananthapuram">Thiruvananthapuram</Option>
+          <Option value="Thrissur">Thrissur</Option>
+          <Option value="Wayanad">Wayanad</Option>
+        </OptGroup>
+      </Select>
```
Comment (vishnuchandrappan) [inline src/components/form-elements/Location.jsx]: Use and `array` and `array.map()` instead

**Review by vishnuchandrappan — CHANGES_REQUESTED** [review]

File: src/components/form-elements/Location.jsx (hunk 2 above)
Comment (vishnuchandrappan) [reply src/components/form-elements/Location.jsx]: Or use `cascader` instead.

Make `states` object similar to 
```js
const states = [
  {
    value: 'Kerala',
    label: 'Kerala',
    children: [
      {
        value: 'Alappuzha',
        label: 'Alappuzha',
        children: [
          {
            value: 'Mararikkulam',
            label: 'Mararikkulam',
          },
          {
            value: 'Kalavoor',
            label: 'Kalavoor',
          },
        ],
      },
      {
        value: 'Kottayam',
        label: 'Kottayam',
        children: [
          {
            value: 'Kanjikkuzhy',
            label: 'Kanjikkuzhy',
          },
          {
            value: 'Vaikkom',
            label: 'Vaikkom',
          },
        ],
      },
    ],
  },
  {
    value: 'Karnataka',
    label: 'Karnataka',
    children: [
      ...
    ],
  },
];
```

Comment (vishnuchandrappan) [reply src/components/form-elements/Location.jsx]: OnChange function gives an array of values.
Its better.

File: src/components/forms/CommunityForm.jsx [hunk 3]
```diff
@@ -93,6 +150,9 @@ export const CommunityForm = () => {
 
     console.log("Received values of form: ", data);
   };
+  function onChange(value){
+  console.log(value);
+  }
```
Comment (vishnuchandrappan) [inline src/components/forms/CommunityForm.jsx]: What does this do ?

Store cheyyande ?

File: src/components/forms/ManagementForm.jsx [hunk 4]
```diff
@@ -233,6 +244,18 @@ export const ManagementForm = () => {
...
+                    rules={[
+                        {
+                            required: true,
+                        },
+                    ]}
+                    label="Second language chosen"
+                    name="second_language"
+                >
+                    <Input placeholder="Eg: Hindi" />
+                </Item>
```
Comment (vishnuchandrappan) [inline src/components/forms/ManagementForm.jsx]: This is OK.
But since we use this in both forms, can be made as a separate component.

**Review by vishnuchandrappan — CHANGES_REQUESTED** [review]

File: src/components/forms/CommunityForm.jsx (hunk 3 above)
Comment (annetajoy) [reply src/components/forms/CommunityForm.jsx]: Function for the diocese option

**Pull request merged**